static/data/data.json

```diff
@@ -278,7 +278,35 @@
       "tags": ["unreal", "angelscript", "automation", "server", "ast", "antlr", "grammar"],
       "description": "A tool to automatically remove server only code from Unreal code bases.",
       "repo": "https://github.com/EmbarkStudios/ServerCodeExciser"
-    }    
+    },
```

Comment:
Should this really list the individual crates, think we generally just lite the project / repository here as that is the "open source project"? Assuming the crates do fit together under the same project.

Think that could be a bit cleaner, and all the individual published crates are visible on crates.io.

Reply:
I seem to remember an issue where the bot would complain if all published crates weren't listed. Only crash-context is published now atm I just wanted to head off the bot, but maybe I'm misremembering.